localstack-core/localstack/services/lambda_/invocation/lambda_models.py

```diff
@@ -625,6 +626,7 @@ class Function:
     provisioned_concurrency_configs: dict[str, ProvisionedConcurrencyConfiguration] = (
         dataclasses.field(default_factory=dict)
     )
+    scaling_configs: dict[str, FunctionScalingConfig] = dataclasses.field(default_factory=dict)
```

Comment:
nit: What naming do you think is most appropriate?
* `scaling_configs`: omitting the function part, assuming it's implicit as part of `Function`
* `function_scaling_configs`: matching the typing and API, similar to `function_url_configs` above

I have a slight preference for the consistency of `function_scaling_configs` in the data model (e.g., easier to search and match to API), but I also like the shorter name of `scaling_configs`. wdyt?